### PR TITLE
Add SNES9x to RAPPS

### PR DESCRIPTION
**Pull request by LothaireCliquennois**
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3467778/SNES9x.txt)

This is my second ever pull request on ROS's repos, tell me if anything goes bad.
Tested it in my fresh VM this morning.

### CODE REVIEW

**Comment by katahiromz** [comment]
Screenshot please.

**Comment by shytzedaka** [comment]
Is it functional? Does it run the games? 
Whats the status on Audio? is it Unknown?

**Comment by LothaireCliquennois** [comment]
> Is it functional? Does it run the games?
> Whats the status on Audio? is it Unknown?

It is functional. I have no problem playing games...
Sound is another story, trying to get Virtualbox's sound working.
Might be good to have the built-in media player to support mp3 or ogg files for testing.

**Comment by LothaireCliquennois** [comment]
> Screenshot please.

image

**Comment by katahiromz** [comment]
```txt
Version = 1.6.0
```
Version is not 1.6.0 but 1.60.

**Comment by BenNottelling** [comment]
> OpenGL was the only change I made to the settings...

Is that needed to have it run?
If so, please add it to the description when addressing the comment by @katahiromz

**Comment by LothaireCliquennois** [comment]
> > OpenGL was the only change I made to the settings...
> 
> Is that needed to have it run?
> If so, please add it to the description when addressing the comment by @katahiromz

It is needed to run, yes.
I adressed the version issue, and I adressed the description to add to the .txt file
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3471280/SNES9x.txt)

File: SNES9x.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = SNES9x
+Version = 1.6.0
```
Comment (katahiromz) [inline SNES9x.txt]:
```suggestion
Version = 1.60
```

**Comment by katahiromz** [comment]
Can you add notice about OpenGL? Thanks.

**Comment by LothaireCliquennois** [comment]
[SNES9x.txt](https://github.com/reactos/rapps-db/files/3478634/SNES9x.txt)
I made a file with all the changes, and sent it yesterday but it looks like I'm too dumb to do a freaking commit, even though it sounds like the first thing to learn in the git workflow.

**Comment by LothaireCliquennois** [comment]
> Now it should be good to go.

And I am a freaking genius lmao I learned how to commit changes to my pull request...
That might seem retarded, but it's a good thing...

File: SNES9x.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[Section]
+Name = SNES9x
+Version = 1.60
+License = GPL
+Description = a portable, freeware Super Nintendo Entertainment System (SNES) emulator. You'll need to change the video renderer to OpenGL in the video settings or in the ini file.
+Size = 2.61 MiB
+Size = 2.61 MiB
```
Comment (katahiromz) [inline SNES9x.txt]: You doubled this line.

File: SNES9x.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Category = 4
+URLSite = http://www.snes9x.com/
+URLDownload = https://sites.google.com/site/bearoso/snes9x/snes9x-1.60-win32.zip?attredirects=0&d=1
+SHA1 = a67156f159405c73be86df49818f9984dfee218d
+CDPath = none
+SizeBytes = 2619607
+[Section.040c]
+Description = Un émulateur portable et gratuit de Super Nintendo Entertainment System (SNES). Vous devrez utiliser OpenGL comme moteur de rendu dans les paramètres vidéo ou dans le fichier .ini.
+[Section.0a]
+[Section.0a]
```
Comment (katahiromz) [inline SNES9x.txt]: Ditto.

**Pull request merged**